### DISCUSSION TITLE
图片如果上下不空行,就无法点击放大

### DISCUSSION THREAD

**Discussion by pendj** [CaiJimmy/hugo-theme-stack#843] [Q&A] [answered]
如题,我使用的最新版(2023/07/20)hugo-theme-stack-starter工程创建的项目,也通过git clone主题的方式试过,都复现了标题中的问题.
![image](https://github.com/CaiJimmy/hugo-theme-stack/assets/112041559/48dff4cf-854e-484f-8f0a-2c016a1a83c8)
像上面这样就可以点击放大或分享.

![image](https://github.com/CaiJimmy/hugo-theme-stack/assets/112041559/e4cbaa2c-f47c-46b2-b320-c8eae26e808d)
像上面这样就无法点击放大或分享,甚至必须得上下两行都有空行.

请问这是bug还是什么? 能不能去掉上下两行空格,否则很影响编写习惯

**Comment by Lovirr** [comment, marked as answer]
图片可以放大是因为调用了photoswipe，本来是不可以放大的。
这个应该算特性？
我一般用的typora编写markdown文件，是自动空行的，所以也没遇到这个问题
如果嫌麻烦，可以在写完文字以后通过全局替换把 `![` 替换成 `回车!]` , `.png)`替换成`.png)回车`
